gratipay/main.py

```diff
@@ -109,10 +109,10 @@
     tell_sentry,
     algorithm['get_response_for_exception'],
 
-    gratipay.set_misc_headers,
+    gratipay.set_version_header,
     authentication.add_auth_to_response,
     csrf.add_token_to_response,
-    cache_static.add_caching_to_response if website.cache_static else noop,
```

Comment:
Ah, right: because now `add_caching_to_response` is scoped to all content, not just static content. Yes?